src/test/java/io/nats/client/api/StreamConfigurationTests.java

```diff
@@ -107,19 +107,23 @@ public void testConstruction() {
         assertEquals(sources.get(0).getExternal(), external);
         assertEquals(sources.get(0).getExternal().hashCode(), external.hashCode());
 
-        List<String> lines = ResourceUtils.dataAsLines("SourceBaseJson.txt");
+        List<String> lines = ResourceUtils.dataAsLines("MirrorsSources.json");
         for (String l1 : lines) {
-            Mirror m1 = new Mirror(l1);
-            assertEquals(m1, m1);
-            assertNotEquals(m1, null);
-            assertNotEquals(m1, new Object());
-            for (String l2 : lines) {
-                Mirror m2 = new Mirror(l2);
-                if (l1.equals(l2)) {
-                    assertEquals(m1, m2);
-                }
-                else {
-                    assertNotEquals(m1, m2);
+            if (l1.startsWith("{")) {
+                Mirror m1 = new Mirror(l1);
+                assertEquals(m1, m1);
```

Comment:
What is the point of this comparison?

Reply:
it's reading the list data in the file line by line, doing individual comparisons. I don't want to look at the line unless it's a full object.